logstash-core/src/main/java/org/logstash/LogstashJavaCompat.java

```diff
@@ -17,80 +17,13 @@
  * under the License.
  */
 
-
 package org.logstash;
 
-import java.io.StringReader;
-import java.util.ArrayList;
-import java.util.Collection;
-import org.codehaus.janino.ClassBodyEvaluator;
-import org.logstash.ackedqueue.io.ByteBufferCleaner;
-
 /**
  * Logic around ensuring compatibility with Java 8 and 9 simultaneously.
```

Comment:
This comment no longer makes sense, maybe the class itself has a misleading name in this context.
The only remaining method is `isJavaAtLeast` which is used only in `org.logstash.instrument.monitors.ProcessMonitor.getCpuLoad()` [coderef](https://github.com/elastic/logstash/blob/0c2645c9c6e181334b85f8414165d4e36c9671a0/logstash-core/src/main/java/org/logstash/instrument/monitors/ProcessMonitor.java#L122)